src/lib/SetPreview.svelte

```diff
@@ -25,6 +26,13 @@
 		displayAbcFields = undefined,
 		headingTy = 'h2'
 	}: Props = $props();
+
+	if (!basePath.startsWith(base)) {
```

Comment:
My attempt at dealing with this being wonky if anyone's already done what I told Robin/assumes they need to.